src/components/Game.js

```diff
@@ -879,9 +885,8 @@ class Game extends React.Component {
   The playbook's data structure would be like ["READ_TOPIC",  "PLAY", "AUDIENCE", "AUDIENCE", "AUDIENCE",   "AUDIENCE", "AUDIENCE", "AUDIENCE"]
   */
   generatePlaybook = () =>{
-    let teamASteps = teams.A.length > 0 ? teams.A.length + 1 : 0;
-    let teamBSteps = teams.B.length > 0 ? teams.B.length + 1 : 0;
-
+    let teamASteps = teams.A.length > 0 ? teams.A.length * 2 : 0;
```

Comment:
Can we discuss the logic here when you are available?

Reply:
sure

Reply:
For the old getRole function. It used switch to manage the memberCnt.
Now I have replay role at the end of every round.
For 1 player, no replay.
For 2 players, only one video can be replayed
For 3 players, two videos can be replayed.
Therefore, the step for 1 is 2, 2 is 4, 3 is 6.
That's the reason why I doubled the steps.

Reply:
Got it. That makes sense to me.